sklearn/linear_model/_base.py

```diff
@@ -501,6 +506,26 @@ def fit(self, X, y, sample_weight=None):
         self : returns an instance of self.
         """
 
+        if self.normalize != "deprecate":
+            if not self.normalize:
+                warnings.warn("'normalize' was deprecated in version 0.24"
+                              " and will be removed in 0.26.", FutureWarning)
```

Comment:
```suggestion
                warnings.warn(
                    "'normalize' was deprecated in version 0.24 and will be removed "
                    "in 0.26.", FutureWarning
                )
```

You will be able to write more lines. It will be useful for the warning below